facet-reflect/src/error.rs

```diff
@@ -1,7 +1,7 @@
 use facet_core::{Characteristic, EnumDef, Field, FieldError, Shape};
 
 /// Errors that can occur when reflecting on types.
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
I also sneakily added this in here, it's very useful to be able to compare what kind of error you have.

Reply:
Sure, that's okay!